docs/docs/developers/tutorials/codealong/first_fees.md

```diff
@@ -0,0 +1,354 @@
+---
+title: All about fees
+sidebar_position: 0
+tags: [fees, accounts, transactions, cli, contracts]
+---
+
+The Aztec network is a privacy preserving layer 2 secured by Ethereum, and for the consensus mechanism to work, Aztec makes use of an asset to pay for transactions called, "Fee juice".
+
+By the end of this tutorial you will...
+- Connect to the Aztec sandbox and/or testnet
+- Use different payment methods to deploy accounts and make transactions via:
+	- various `aztec-wallet` CLI commands
+	- the `aztec.js` library
+- Understand the pros/cons of different payment methods
+
+
+## Background
+
+As a quick summary of required knowledge:
+
+(**TODO: Each of these as snippets**)
+
+**The PXE**
+The PXE is a client-side key manager, private contract storage, and Private eXecution Environment for private transactions. It interfaces with the Aztec network, and can be used via its json RPC client. A PXE is a core part of an Aztec wallet and Sandbox, but can be decoupled and run independently.
+
+**An Aztec node**
+An Aztec node is a prover/sequencer that is part of a decentralised Aztec network. The Aztec testnet rolls up to Ethereum Sepolia.
+
+**The Aztec Sandbox**
+The Aztec Sandbox runs a local environment for rapid development, it includes: an Ethereum node, an Aztec node, and PXE.
+
+## Connect to the network
+
+We'll go through both the `aztec-wallet` cli wallet and the `aztec.js` library, to a local Aztec network (in the sandbox) and the Aztec testnet.
+
+### Tools
+
+**TODO: as snippet**
+To use Aztec's suite of tools you'll need to:
+ - [Get docker](https://docs.docker.com/get-started/get-docker/)
+ - Run `bash -i <(curl -s https://install.aztec.network)`
+
+Test the cli wallet with: `aztec-wallet --version`
+
+By default the sandbox runs everything including a pxe. For this tutorial, and more realistically when using testnet, we will be using PXEs client side. Eg as part of a wallet.
+
+Start the sandbox (L1, L2, but not the PXE) via: `NO_PXE=true aztec start --sandbox`
+
+### Specifying the network URL in commands
+
+When using `aztec-wallet`, the Aztec network node to connect to can be specified in each relevant command:
+
+```bash
+aztec-wallet --node-url <string> ...
+```
+
+The string currently defaults to the the local sandbox: `http://host.docker.internal:8080`, or can be a bootnode url.
```

Comment:
does it default to this or localhost:8080? or is it functionally the same thing?

Reply:
I saw the host in one file, and localhost in another, so it would be the same if the container is exposing it locally. (Or localhost if connecting things within the docker container).
Just reading an internal thread about sandbox/testnet releases, and I'm not sure if this can correctly be pointed to testnet now (might just be the default in the build?).

Will get clarity once it settles and update once we can test it.